lib/feed.js

```diff
@@ -1,109 +1,101 @@
+var RESPONSE_GRACE_TIME = 5000;
 
-var FEED_PARAMETERS   = ['since', 'limit', 'feed', 'heartbeat', 'filter', 'include_docs', 'view', 'style', 'conflicts', 'attachments', 'att_encoding_info'];
+var FEED_PARAMETERS = ['since', 'limit', 'feed', 'heartbeat', 'filter', 'include_docs', 'view', 'style', 'conflicts', 'attachments', 'att_encoding_info'];
 
 var EventEmitter = events.EventEmitter2 || events.EventEmitter;
 
-
 util.inherits(Feed, EventEmitter);
-function Feed (opts) {
+function Feed(opts) {
   var self = this;
   EventEmitter.call(self);
 
-  opts = opts || {}
+  opts = opts || {};
 
   self.feed = 'continuous';
-  self.heartbeat         = opts.heartbeat || DEFAULT_HEARTBEAT;
+  self.heartbeat = opts.heartbeat || DEFAULT_HEARTBEAT;
   self.max_retry_seconds = opts.max_retry_seconds || DEFAULT_MAX_RETRY_SECONDS;
   self.inactivity_ms = null;
   self.initial_retry_delay = opts.initial_retry_delay || INITIAL_RETRY_DELAY;
   self.response_grace_time = opts.response_grace_time || RESPONSE_GRACE_TIME;
 
   self.headers = {};
-  self.request = opts.request || {} // Extra options for potentially future versions of request. The caller can supply them.
+  self.request = opts.request || {}; // Extra options for potentially future versions of request. The caller can supply them.
 
   self.since = 0;
-  self.is_paused = false
-  self.caught_up = false
+  self.is_paused = false;
+  self.caught_up = false;
   self.retry_delay = self.initial_retry_delay;
 
   self.query_params = {}; // Extra `req.query` values for filter functions
 
-  if(typeof opts === 'string')
-    opts = {'db': opts};
+  if (typeof opts === 'string') { opts = {'db': opts}; }
 
   Object.keys(opts).forEach(function(key) {
-    if (typeof self[key] !== 'function')
-      self[key] = opts[key];
-  })
+    if (typeof self[key] !== 'function') { self[key] = opts[key]; }
+  });
 
-  self.pending = { request     : null
-                 , activity_at : null
-                 };
+  self.pending = { request: null,
+    activity_at: null
+  };
 } // Feed
 
 Feed.prototype.start =
 Feed.prototype.follow = function follow_feed() {
   var self = this;
 
-  self.db = self.db || self.url || self.uri
-  delete self.url
-  delete self.uri
+  self.db = self.db || self.url || self.uri;
+  delete self.url;
+  delete self.uri;
 
-  if(!self.db)
-    throw new Error('Database URL required');
+  if (!self.db) { throw new Error('Database URL required'); }
 
-  if (self.db.match(/\/_db_updates$/))
-    self.is_db_updates = true;
+  if (self.db.match(/\/_db_updates$/)) { self.is_db_updates = true; }
 
-  if(self.is_db_updates)
-    delete self.since;
+  if (self.is_db_updates) { delete self.since; }
 
-  if(self.feed !== 'continuous' && self.feed !== 'longpoll')
-    throw new Error('The only valid feed options are "continuous" and "longpoll"');
+  if (self.feed !== 'continuous' && self.feed !== 'longpoll') { throw new Error('The only valid feed options are "continuous" and "longpoll"'); }
```

Comment:
Here's another example where it is harder to read than before because of the line collapse.